tests/kuttl/test-suites/default/common/cleanup-errors.yaml

```diff
@@ -57,6 +57,13 @@ metadata:
   - openstack.org/watcherapi
   name: watcher-kuttl-api
 ---
+apiVersion: watcher.openstack.org/v1beta1
```

Comment:
I would move this to the bottom of the file, so we can group each controller, i.e WatcherAPI+its deployment, service,etc, WatcherApplier+its deployment, WatcherDecistionEngine+its deployment. I think it'll make easier for you as you add more resources and have to tweak the test files

Reply:
ack